tools/github-event-processor/Azure.Sdk.Tools.GitHubEventProcessor/EventProcessing/IssueProcessing.cs

```diff
@@ -0,0 +1,489 @@
+        // 1. If the rule is based upon an Action (event), verify the rule action matches the action
+        // 2. If #1 is true, verify all of the conditions
+        // 3. If using an IssueUpdate, it'll be a ref parameter (if non-async), otherwise the funtion will
+        //    return an updated IssueUpdate
+        /// <summary>
+        /// Initial Issue Triage
+        /// Trigger: issue opened
+        /// Conditions: Issue has no labels
+        ///             Issue has no assignee
+        /// Resulting Actions:
+        ///     Evaluate the user that created the issue:
+        ///         IF creator is NOT an Azure SDK team owner
+        ///         AND is NOT a member of the Azure organization
+        ///         AND does NOT have write permission
+        ///         AND does NOT have admin permission:
+        ///             Add "customer-reported" label
+        ///             Add "question" label
+        ///     Query AI label service for label suggestions:
+        ///     IF labels were predicted:
+        ///         Assign returned labels to the issue
+        ///         Add "needs-team-attention" label to the issue
+        ///         IF service label is associated with an Azure SDK team member:
+        ///             IF a single team member:
+        ///                 Assign team member to the issue
+        ///             ELSE
+        ///                 Assign a random team member from the set to the issue
+        ///                 Add a comment mentioning the other team members from the set
+        ///                 Add comment indicating issue was routed for assistance: "Thank you for your feedback.  Tagging and routing to the team member best able to assist."
+        ///         ELSE (service label is not associated with an Azure SDK team member)
+        ///             Add "CXP Attention" label to the issue
+        ///             Create a comment mentioning(content from .NET rule #30)
+        ///     ELSE (labels were not predicted)
+        ///         Add "needs-triage" label to the issue
+        /// </summary>
+        /// <param name="gitHubEventClient">Authenticated GitHubEventClient</param>
+        /// <param name="issueEventPayload">IssueEventGitHubPayload deserialized from the json event payload</param>
+        public static async Task InitialIssueTriage(GitHubEventClient gitHubEventClient, IssueEventGitHubPayload issueEventPayload)
+        {
+            if (gitHubEventClient.RulesConfiguration.RuleEnabled(RulesConstants.InitialIssueTriage))
+            {
+                if (issueEventPayload.Action == ActionConstants.Opened)
+                {
+                    // If there are no labels and no assignees
+                    if ((issueEventPayload.Issue.Labels.Count == 0) && (issueEventPayload.Issue.Assignee == null))
+                    {
+                        // JRS - IF creator is NOT an Azure SDK team owner
```

Comment:
Aren't these JRS comments todo's? If so lets make sure we have follow-up issues filed to track them.

Reply:
I replaced JRS with TODO and created an [issue](https://github.com/Azure/azure-sdk-tools/issues/5743) to update the Initial Issue Triage rule when they are. I cleaned up the function, itself, of the TODOs.